### DISCUSSION TITLE
Panel 2 Q/A

### DISCUSSION THREAD

**Discussion by cmbarton** [cssi-pi-community/2022-meeting#8] [Panelist Q/A for all discussions]
### Post a question for panelists in a comment responding to this discussion thread.

**Comment by sunbiz** [comment]
To point out a group that is working continuously for research metadata and tools is under the Research Data Alliance - https://www.rd-alliance.org/groups/metadata-standards-directory-working-group.html

**Comment by erjank** [comment]
Should data management plans be enforced? If not, why require them? If so, is there work on a "restorative justice" approach that holds folks accountable while transforming data management practices?

**Reply by cmbarton** [reply]
They **should** be if they are to have any meaning at all. There need to be _incentives_ for DMP compliance, like requiring past DMP reports in order to receive funding or required with a proposal, like a prior research section.

**Reply by munkm** [reply]
> If so, is there work on a "restorative justice" approach

Ooooh interesting idea! What do you mean here? That if a past project isn't compliant with DMP that they have pathways to become compliant?

**Reply by cmbarton** [reply]
In response to the NSF PM, compliance with a DMP should be in the context of the accepted DMP. It can be reported in the section on prior research. Proposers should _at least_ need to account for how they have complied with their own DMPs

**Reply by erjank** [reply]
Yeah, if say a spot check reveals major gaps  in how records are being organized and maintained, rewarding a team for fixing it and providing support to other groups for fixing similar problems for related teams. Or something!

**Reply by erjank** [reply]
Discussion from table 22 of self-reporting DMP compliance (or shortcomings, or challenges, etc) in annual/final reports for projects

**Reply by munkm** [reply]
> Yeah, if say a spot check reveals major gaps in how records are being organized and maintained, rewarding a team for fixing it and providing support to other groups for fixing similar problems for related teams. Or something!

I love that idea! Rewarding the process of correcting poor(er) practices, rather than just rewarding the people who start doing it right in the first place.

**Comment by dtarb** [comment]
On the topic of metadata - standards such as Dublin Core and various ISO standards apply.  For the Hydrology domain we have suggested the "Resource data model" used for metadata for hydrologic resources.  There are approaches here that could be generalized for other domains.  The trick is keeping it simple, not requiring too much initially but enabling metadata to be added with time.  There is information at https://help.hydroshare.org/hydroshare-resources/ and http://dx.doi.org/10.1111/1752-1688.12363.

**Reply by cmbarton** [reply]
For computational modeling, a recently created organization, the Open Modeling Foundation, is aimed at developing FAIR standards and incentives for FAIR practice. [https://openmodelingfoundation.github.io](https://openmodelingfoundation.github.io)
